cockroachdb/templates/job.init.yaml

```diff
@@ -97,57 +99,32 @@ spec:
         - name: cluster-init
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
           imagePullPolicy: {{ .Values.image.pullPolicy | quote }}
-          # Run the command in an `while true` loop because this Job is bound
-          # to come up before the CockroachDB Pods (due to the time needed to
-          # get PersistentVolumes attached to Nodes), and sleeping 5 seconds
-          # between attempts is much better than letting the Pod fail when
-          # the init command does and waiting out Kubernetes' non-configurable
-          # exponential back-off for Pod restarts.
           # Command completes either when cluster initialization succeeds,
           # or when cluster has been initialized already.
           command:
           - /bin/bash
           - -c
-          - >-
+          - |
             {{- if $isClusterInitEnabled }}
-              initCluster() {
-                while true; do
```

Comment:
Removing this loop can cause issues, what if the init was not successful in first attempt. Job will fail and cockroachdb will not come up.

Reply:
`cockroach init` already runs in a loop. I think this was changed since the Job was created and the Job was never updated..